app/src/renderer/vuex/modules/blocks.js

```diff
@@ -18,7 +18,10 @@ export default ({ node }) => {
     loading: false,
     error: null,
     peers: [],
-    blocks: []
+    blocks: [],
+    block: {
```

Comment:
what is `block` ? is it the latest one ? In any case, this should go with a comment or variable renaming

Reply:
added comment - done!